### PR TITLE
Implemented new quick action buttons on user recipe page

### PR DESCRIPTION
**Pull request by CamronOlsen**

### CODE REVIEW

File: client/public/static/js/account/my_recipes.js [hunk 1]
```diff
@@ -339,3 +359,167 @@ $(document).ready(function () {
...
+  }
+
+  const userId = await utils.getUserIdFromUsername(username);
+  if (!userId) {
+    console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    return;
+  }
+
+  window.location = "/account/update_recipe?userRecipeName=" + recipeName;
```
Comment (Thompson632) [inline client/public/static/js/account/my_recipes.js:378]: Change to be: 

`/account/update_recipe?userRecipeName=${recipeName}`

File: client/public/static/js/account/my_recipes.js [hunk 2]
```diff
@@ -339,3 +359,167 @@ $(document).ready(function () {
...
+}
+
+async function deleteRecipe(recipeName) {
+  const username = utils.getUserNameFromCookie();
```
Comment (Thompson632) [inline client/public/static/js/account/my_recipes.js:382]: The checking of user name and user id is duplicated a bunch here's should create a method to handle

File: client/public/static/js/account/my_recipes.js [hunk 3]
```diff
@@ -252,6 +252,15 @@ $(document).ready(function () {
                           <a href="/recipes/recipe_details?source=${recipeSource}&sourceUrl=${recipeSourceUrl}&uri=${recipeUri}">
                               <img src="${recipeImage}" alt="${recipeName}" title="View more about ${recipeName}">
                           </a>
+                            <div class="recipe-dropdown">
```
Comment (Thompson632) [inline client/public/static/js/account/my_recipes.js:255]: Is this functionality specifically only to a single recipe? I could see us doing this exactly functionality in the recipes view as well but no action yet

**Review by Thompson632 — CHANGES_REQUESTED** [review]

**Comment by Thompson632** [comment]
I'm not going to be able to view until later in the week could you add a screenshot or two of the updates pls

File: client/public/static/js/account/my_recipes.js (hunk 3 above)
Comment (CamronOlsen) [reply client/public/static/js/account/my_recipes.js:255]: Yea, the dropdowns are on each and every separate recipe.

File: client/public/static/js/account/my_recipes.js (hunk 1 above)
Comment (CamronOlsen) [reply client/public/static/js/account/my_recipes.js:378]: I couldn't get this to work. RecipeName is not being seen as a variable in the string when done like this for me

**Comment by CamronOlsen** [comment]
New dots on images:
image
Update and Delete on self made recipes:
image
Unfavorite on all other recipes:
image

**Pull request merged**